packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -2859,27 +2860,32 @@ function updateDehydratedSuspenseComponent(
       // This boundary is in a permanent fallback state. In this case, we'll never
       // get an update and we'll never be able to hydrate the final content. Let's just try the
       // client side render instead.
-      let digest, message, stack;
+      let digest: ?string;
```

Comment:
I hit a weird Flow error that only repro:ed on CI and not locally. I thought they were supposed to be the same version. It considers it empty even though it's always initialized.